Cargo.toml

```diff
@@ -16,6 +16,10 @@ embedded-hal = { version = "0.2.3", features = ["unproven"] }
 nb = "0.1"
 riscv = "0.5"
 
+[dependencies.void]
+default-features = false
+version = "1.0.2"
```

Comment:
Maybe we could use `Infallible` in `core::convert` by now, it does not require another dependency.